imitation/util/trainer.py

```diff
@@ -56,6 +53,33 @@ def init_trainer(env_id, use_gail=False,
                               scale=discrim_scale, **reward_kwargs)
     discrim = discrim_net.DiscrimNetAIRL(rn, **discrim_kwargs)
 
-  trainer = Trainer(env, gen_policy, discrim,
-                    expert_policies=expert_policies, **trainer_kwargs)
+  ro_paths = util.get_dump_paths(
```

Comment:
This seems more complicated than it needs to be. Why don't we just take a path to a list of rollout files?

Right now we're assuming a particular directory structure, making `train` very coupled to `data_collect`. What if I want to use e.g. human demonstrations, or my own training algorithm?

Reply:
I'm starting to think that it would be best for Sacred config to take only a single npz file. Data collect can combine expert rollouts from several timesteps into a single npz if necessary.

Reply:
I'm fine (and weakly in favour) of a single npz file. But we could support multiple npz files and still avoid assuming a particular directory structure, by having Sacred config take a list of rollouts or a rollout glob path.